### PR TITLE
[doc] multiple corrections to "Summary of the tasks"

### PR DESCRIPTION
**Pull request by stas00**
- improve readability - typos, punctuation, clearer sentences, numerical bullets where needed
- fix incorrect script names
- add missing scripts and add hyper links where missing
- a minor code improvement to improve understanding at the very last section (replaced output ids with translated text)

One thing I didn't know how to fix is the outdated referance to a script that no longer exists:

> If you would like to fine-tune a model on a summarization task, you may leverage the ``examples/summarization/bart/run_train.sh`` (leveraging pytorch-lightning) script.

@sgugger

### CODE REVIEW

File: docs/source/task_summary.rst [hunk 1]
```diff
@@ -709,13 +725,12 @@ following array should be the output:
 Summarization
 ----------------------------------------------------
 
-Summarization is the task of summarizing a text / an article into a shorter text.
+Summarization is the task of summarizing a document or an article into a shorter text.
 
 An example of a summarization dataset is the CNN / Daily Mail dataset, which consists of long news articles and was created for the task of summarization.
-If you would like to fine-tune a model on a summarization task, you may leverage the ``examples/summarization/bart/run_train.sh`` (leveraging pytorch-lightning) script.
+If you would like to fine-tune a model on a summarization task, you may leverage the ``examples/summarization/bart/run_train.sh`` (XXX: wrong script) (pytorch-lightning) script.
```
Comment (sgugger) [inline docs/source/task_summary.rst:731]: @sshleifer There is no script there indeed. Do you know where the summarization examples are? Didn't find any named like this in the seq2seq folder.

**Review by sgugger — APPROVED** [review]
Great work! We just have to fin that missing script ;-)

File: docs/source/task_summary.rst (hunk 1 above)
Comment (sshleifer) [reply docs/source/task_summary.rst:731]: https://github.com/huggingface/transformers/blob/72add6c98f2c0607f088fa0c78d40f11e2efa4c4/examples/seq2seq/README.md#L99

MT+Summary are both in `examples/seq2seq/finetune.py`, would be good to link to that README which has info on how to use your own data/how to distill etc.

**Comment by sgugger** [comment]
Good to go thanks. (side-note: we tend to default to two underscores for links as otherwise, sphinx will always associate the description to the same link).

**Comment by stas00** [comment]
> side-note: we tend to default to two underscores for links as otherwise, sphinx will always associate the description to the same link

I will know now - thank you: fixed here https://github.com/huggingface/transformers/pull/6541

**Pull request merged**